jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/HttpRequestDispatch.java

```diff
@@ -122,7 +123,11 @@ private void honourMaxKeepAliveRequests() {
             boolean reportedError = false;
 
             if (error != null) {
-                if (!(error instanceof OverloadException || error instanceof BindingNotFoundException)) {
+                if (error instanceof EofException) {
+                    log.log(Level.FINE,
```

Comment:
Why so many line breaks?

Reply:
I also suggest to static import FINE and WARNING

Reply:
Because I follow the line limit strictly.

Reply:
At which column?